net/dhcp6/files/patch-common.h

```diff
@@ -1,11 +1,10 @@
 --- common.h.orig	2007-03-21 09:52:57 UTC
 +++ common.h
-@@ -155,7 +155,7 @@ extern char *in6addr2str __P((struct in6
- extern int in6_addrscopebyif __P((struct in6_addr *, char *));
- extern int in6_scope __P((struct in6_addr *));
- extern void setloglevel __P((int));
--extern void dprintf __P((int, const char *, const char *, ...));
-+extern void d_printf __P((int, const char *, const char *, ...));
```

Comment:
Why are you removing this change? d_printf is being used on FreeBSD ports for a reason... Are you applying your changes on top of upstream code? Or on top of FreeBSD ports code with patches applied?